variables.tf

```diff
@@ -21,6 +21,12 @@ variable "edge_zone" {
   description = "(Optional) Specifies the Edge Zone within the Azure Region where this Public IP and Load Balancer should exist. Changing this forces new resources to be created."
 }
 
+variable "frontend_ip_zones" {
+  type        = list(string)
```

Comment:
Could we change the type to `set(string)` here?

Reply:
@lonegunmanb updated